apps/association_result_annotation/dxapp.json

```diff
@@ -1,120 +1,144 @@
 {
   "name": "association_result_annotation",
   "title": "Annotate GWAS, PheWAS Associations",
-  "summary": "association_result_annotation",
+  "summary": "This app increases the depth of annotation information in an input GWAS tsv file.",
   "dxapi": "1.0.0",
-  "version": "0.0.1",
-  "categories": [],
+  "version": "1.0.0",
+  "details": {
+    "whatsNew": "*1.0.0: updated to ubuntu 14.04, added patterns/help to input/output spec, added regionalOptions/assetDepends;",
```

Comment:
"whatsNew": "\*1.0.0: updated to ubuntu 14.04, added patterns/help to input/output spec, added regionalOptions/assetDepends;",
->
"\* 1.0.0 First version of the Annotate GWAS, PheWAS Associations app."